ci/unit/test_main.sh

```diff
@@ -7,6 +7,27 @@ set -ex
 # JENKINS-52781 tar function is breaking symlinks
 rm -rf test_results
 mkdir test_results
+
+# Check if this is a Bulleye stage
+USE_BULLSEYE=
+if [ -n "${STAGE_NAME:?}" ]; then
+  case $STAGE_NAME in
+    *Bullseye**)
+      USE_BULLSEYE="true"
+      ;;
+  esac
+fi
+
+: "${BULLSEYE:=}"
+
+if [ -n "$USE_BULLSEYE" ];then
+  rm -rf bullseye
+  mkdir -p bullseye
+  tar -C bullseye --strip-components=1 -xf bullseye.tar
+else
+  BULLSEYE=
```

Comment:
Shouldn't be needed as a default value of `false` is already set.
```suggestion
```

Reply:
BULLSEYE is different than USE_BULLSEYE.

If we are in a NOT_BULLSEYE, I need to clear the BULLSEYE environment variable if it exists instead of passing it on to the nodes.